src/component-select/component-select.scss

```diff
@@ -10,30 +10,32 @@
 
   button {
     margin-left: $spacing;
-
-    &:hover {
-      color: green;
-    }
   }
 
   .page-area {
     display: flex;
     align-items: center;
 
-    button {
+    .edit-page-name {
       background-color: transparent;
       border: none;
-      font-size: 20px;
+      font-size: 15px;
       padding: 0;
+      &:hover {
+        color: $primary-orange;
+      }
     }
 
-    .page {
+    .page, .page input[type='text'] {
       color: $gray4;
       display: inline-block;
       font-size: 20px;
       font-style: italic;
       font-weight: 100;
       line-height: 28px;
+      &.page input[type='text'] {
```

Comment:
Shouldn't line 36 be just this?
input[type="text"] {

Reply:
It should, yes. My mistake.